README.md

```diff
@@ -32,6 +32,7 @@ spec:
   baseURL: https://gitlab.com/
   credentials:
     source: Secret
+    method: PrivateToken
```

Comment:
```suggestion
    method: PersonalAccessToken
```

Reply:
I used this naming referring to the dependency https://github.com/xanzy/go-gitlab/blob/main/gitlab.go#L67 that initializes the source but you are right that the source of truth is gitlab itself so I will rename it: https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html